tools/flash-shelly.py

```diff
@@ -293,11 +293,12 @@ def parse_local_file(self):
       s.close()
       with zipfile.ZipFile(self.local_file, "r") as zfile:
         for name in zfile.namelist():
-          if re.search(r'\.json$', name) is not None:
+          if re.search(r'manifest.json$', name) is not None:
```

Comment:
if name.endswith('manifest.json')

Reply:
not sure is will search subdirectories using that, let me test.

Reply:
ok works, done